PFS-SPS-PRU030006/README.md

```diff
@@ -0,0 +1,6 @@
+# SPS Software Test Cases (PFS-PRU-300005)
```

Comment:
PFS-PRU-300005 -> PFS-SPS-PRU300006

Reply:
Thanks. This was a cut and paste error. I've corrected the document ID and description.